RunWorkFlow_CSHORE.py

```diff
@@ -35,8 +36,8 @@ def master_CSHORE_run(inputDict):
     """
 
     version_prefix = inputDict['version_prefix']
-    endTime = inputDict['endTime']
-    startTime = inputDict['startTime']
+    endTime = inputDict['end_date']
```

Comment:
resolved - also resolved in example yaml file.

Reply:
sorry nevermind, I was looking at the wrong file.

I renamed them in my new CSHORE stuff so that anyone debugging won't be confused between inputDict['startTime'] and the startTime inside of frontBackCSHORE.

If you give it a big range of days, then the startTime and endTime inside of frontBackCSHORE will be different then those in the inputDict - at least in the days in the middle.  renaming it to end_date and start_date separates the two variables.

I'll change it back if you want me to.